app/controllers/idv/hybrid_mobile/entry_controller.rb

```diff
@@ -0,0 +1,80 @@
+module Idv
+  module HybridMobile
+    # Controller responsible for taking a `document-capture-session` UUID and configuring
+    # the user's Session to work when they're forwarded on to document capture.
+    class EntryController < ApplicationController
+      include HybridMobileConcern
+
+      before_action :track_document_capture_session_id_usage
+      before_action :validate_document_capture_session_id
+      before_action :validate_document_capture_user_id
+
+      def show
+        redirect_to idv_hybrid_mobile_capture_doc_url
```

Comment:
Redirect to the new Document Capture controller
```suggestion
        redirect_to idv_hybrid_mobile_document_capture_url
```